Add option to the inventory plugin to retrieve a list of ansible roles enabled within foreman

stuwilkins opened the pull request:
This addition to the inventory plugin pulls the defined ansible roles from foreman and sets the variable `foreman_ansible_roles`

The option is set via the `wants_ansible_roles` setting.

stuwilkins committed:
Add option to inventory plugin to pull ansible roles.

Added option to inventory plugin to pull the list of
ansible roles defined by foreman for a given host. The list
of ansible roles are stored in the foreman_roles variable.

stuwilkins committed:
Rename inventory plugin variable

Rename inventory plugin variable to be foreman_ansible_roles to be
consistent with how thw variable is defined for plays within
foreman.

mdellweg commented:
This repository contains so much more than just the inventory plugin. Please drop the name of the component somewhere in the title of this issue and the commit messages.

stuwilkins commented:
@mdellweg Thanks for the prompt reply, and thanks for the work on this collection, we use much more than just the plugin but I added this to help with using Tower and Foreman together. 

I changed the commit messages and also the PR. Please let me know if you need anything else.

mdellweg commented:
That is much better now. And i hope people more familiar with the inventory than me will be triggered by this. ;)
